src/transformState.js

```diff
@@ -5,7 +5,21 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
```

Comment:
function should return state

Reply:
should modify the state, doing the following:

- add two properties to the state
- then remove keys bar and hello from the state
- and finally add another one property to the state